### DISCUSSION TITLE
NotNullViolation when trying to upload CSV

### DISCUSSION THREAD

**Discussion by coder2000** [Shopify/maintenance_tasks#774] [Q&A] [answered]
I'm trying to upload a CSV file to a task and I keep getting an error that `record_id` isn't set on the ActiveStorage attachment. Is there something I am missing in setting up the gem?

```
PG::NotNullViolation: ERROR:  null value in column "record_id" of relation "active_storage_attachments" violates not-null constraint
DETAIL:  Failing row contains (f89c625a-e056-4bf3-9efa-6b7fcb839825, csv_file, MaintenanceTasks::Run, null, ce1d3630-2e89-4bc1-8fa8-6b8f30c33a8b, 2023-02-19 02:04:13.642059).
```

**Comment by etiennebarrie** [comment, marked as answer]
It seems like you're using PostgreSQL with UUIDs, is that correct? I don't think we support it out of the box but we've had some interest in #482. If you're able to contribute support that would be great.
